src/components/NewDraftWizard/WizardComponents/WizardSavedObjectsList.js

```diff
@@ -90,31 +93,37 @@ const WizardSavedObjectsList = ({ submissions }: WizardSavedObjectsListProps): R
 
   return (
     <div className={classes.objectList}>
-      {groupedSubmissions.map(group => (
-        <Box pt={0} key={group.submissionType}>
-          <CardHeader
-            title={`Submitted ${displayObjectType(objectType)} ${displaySubmissionType(group)}`}
-            titleTypographyProps={{ variant: "inherit" }}
-            className={classes.cardHeader}
-          />
-          <List aria-label={group.submissionType}>
-            {group.submittedItems.map(item => (
-              <ListItem key={item.accessionId} className={classes.objectListItem}>
-                <ListItemText className={classes.listItemText} primary={item.accessionId} />
-                <ListItemSecondaryAction>
-                  <WizardSavedObjectActions
-                    submissions={submissions}
-                    objectType={objectType}
-                    objectId={item.accessionId}
-                    submissionType={group?.submissionType}
-                    tags={item.tags}
+      {groupedSubmissions &&
+        groupedSubmissions.map(group => (
+          <Box pt={0} key={group.submissionType}>
+            <CardHeader
+              title={`Submitted ${displayObjectType(objectType)} ${displaySubmissionType(group)}`}
+              titleTypographyProps={{ variant: "inherit" }}
+              className={classes.cardHeader}
+            />
+            <List aria-label={group.submissionType}>
+              {group.submittedItems.map(item => (
+                <ListItem key={item.accessionId} className={classes.objectListItem}>
+                  <ListItemText
+                    className={classes.listItemText}
+                    primary={getItemPrimaryText(item)}
+                    secondary={item.accessionId}
+                    data-testid={item.schema}
```

Comment:
would prefer if we have some like `data-schema` instead of `data-testid` as it makes more sense
the same goes for the other instances of this

Reply:
or `data-schemaType`

Reply:
I changed it to `data-schema`